docs/manual/common/concepts/DevelopmentEnvironmentOverview.md

```diff
@@ -6,11 +6,11 @@ Setting up a Lagom development environment takes only minutes using an sbt templ
 
 * The infrastructure necessary to run and test your services locally:
     
-    * A Kafka server for handling messages. 
+    * A [[Kafka server|KafkaClient]] for handling messages. 
```

Comment:
Sorry, I visually searched for these files. I should have used file search. I totally missed the ones in the common/devmode folder. I've updated the reference.